components/molecules/CallToAction.js

```diff
@@ -13,7 +13,10 @@ export function CallToAction(props) {
       <div className="layout-container pb-10 pt-10 text-xs md:text-base">
         <h2>{props.title}</h2>
         <div className="flex flex-col lg:grid lg:grid-cols-2 lg:gap-24 gap-5">
-          <p>{props.description}</p>
+          <p>
+            {props.descriptionP1} <br />
+            <br /> {props.descriptionP2}
```

Comment:
Part of Government of Canada design guidelines is to never use <br> tags

Reply:
Humm ok sounds good. ill just put some mb-4

Reply:
My comment on how to use the single description prop for multi-line text should address this as well.